### PR TITLE
fix some more command->normal mode renames

### PR DESCRIPTION
**Pull request by bronson**
All style, very little substance.

### CODE REVIEW

File: spec/motions-spec.coffee [hunk 1]
```diff
@@ -22,9 +22,9 @@ describe "Motions", ->
     editor.normalModeInputView.editorElement.getModel().setText(key)
 
   submitNormalModeInputText = (text) ->
-    commandEditor = editor.normalModeInputView.editorElement
-    commandEditor.getModel().setText(text)
-    atom.commands.dispatch(commandEditor, "core:confirm")
+    normalEditor = editor.normalModeInputView.editorElement
+    normalEditor.getModel().setText(text)
+    atom.commands.dispatch(normalEditor, "core:confirm")
```
Comment (jacekkopecky) [inline spec/motions-spec.coffee]: `normalEditor` sounds so generic; maybe `normalModeInputEditor`? Or `inputEditor`?

Comment (jacekkopecky) [inline spec/motions-spec.coffee]: Same below in "using search history".

**Comment by jacekkopecky** [comment]
I'll wait with my rebases until this one gets merged, if that's all right. 8-)

**Comment by bronson** [comment]
Agreed, inputEditor seems nice and readable.  Updated!

I also agree with avoiding rebases until things settle down.  That said, I already did vim-mode-next...  everything seems OK.   https://github.com/bronson/vim-mode-next/commit/80ad67c8c67bdd9c92cfbdda2267c923c03baeee

**Comment by maxbrunsfeld** [comment]
:eyes: Thanks!

**Pull request merged**